src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 16px;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+
+    &:nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg);
```

Comment:
The selector '.stars__star:nth-child(-n + 4)' will always apply the active star image to the first four stars, which is not flexible if you want to represent different star ratings. Consider using a modifier class that can be applied to the stars based on the rating. For example, you can have classes like '.stars--1', '.stars--2', etc., to represent different numbers of active stars.